src/get-started/install.md

```diff
@@ -45,8 +45,8 @@ use
 
 ```shell
 add-apt-repository ppa:exonum/rocksdb
-apt-get install build-essential libsodium-dev libsnappy-dev \
-    librocksdb5.17 pkg-config libprotobuf-dev protobuf-compiler
+apt-get install build-essential libsodium-dev libsnappy-dev libssl-dev \
+    librocksdb5.17 pkg-config
```

Comment:
```suggestion
    librocksdb5.17 pkg-config clang-7 lldb-7 lld-7
```

Reply:
@ivan-ochc are you sure that the meta package `build-essential` doesn't contain these packages ?

Reply:
@aleksuss, yes, `build-essential` doesn't contain these packages.

Reply:
@Maria-Nosyk, @aleksuss , seems, that we need only  `pkg-config, libssl-dev, clang-7, lldb-7, lld-7` and protobuf related packages.